assets/performanceprofile/configs/99-netdev-rps.rules

```diff
@@ -1 +1,12 @@
+# We should apply the rule on both virtual and physical interfaces of the host, because the RPS mask that would be consulted,
```

Comment:
Why physical devices? The traffic for physical devices is handled via RSS and real interrupts and we already control the affinity for those. RPS is only needed for host side veth devices to fix the pod to host traffic.